tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationControllerBuilder.cs

```diff
@@ -14,16 +17,21 @@ internal sealed class OrganisationControllerBuilder
     {
         private IOrganisationRepository _organisationRepository;
         private ICreateOrganisationService _createOrganisationService;
+        private ClaimsPrincipal ClaimsPrincipal;
 
-        private OrganisationControllerBuilder()
+        private OrganisationControllerBuilder(Guid primaryOrganisationId)
         {
             _createOrganisationService = Mock.Of<ICreateOrganisationService>();
             _organisationRepository = Mock.Of<IOrganisationRepository>();
+            ClaimsPrincipal = new ClaimsPrincipal(new ClaimsIdentity(new[]
+            {
+                new Claim("primaryOrganisationId", primaryOrganisationId.ToString())
+            }, "mock"));
```

Comment:
I think the formatting looks a bit odd here. I'd probably be tempted to format it like this:

```c#
_claimsPrincipal = new ClaimsPrincipal(
    new ClaimsIdentity(new[] { new Claim("primaryOrganisationId", primaryOrganisationId.ToString()) }, 
    "mock"));
```
or this:

```c#
_claimsPrincipal = new ClaimsPrincipal(
    new ClaimsIdentity(new[]
    {
        new Claim("primaryOrganisationId", primaryOrganisationId.ToString())
    }, 
    "mock"));
```

Reply:
👍